src/utils/web3.ts

```diff
@@ -43,7 +43,7 @@ export function getNetworkConfigObject(node: any): NetworkObject {
     name: node.chain,
     symbol: node.nativeCurrency.symbol,
     chainId: node.chainId,
-    urlList: [node.providerUri]
+    urlList: node.rpc
```

Comment:
typings should have caught this but `const networkConfig` should be `const networkConfig: NetworkObject`, then TypeScript would correctly complain here as `urlList` is defined as an array at top of file in `NetworkObject`.

Reply:
but `urlList` is an array, just now we get the array from `EthereumListsChain` and no need to be created over here as it was before

Reply:
Ah you're right, my bad. Did not see that node.rpc is an array too